### ISSUE TITLE
BuildBuddy onprem (Docker) >2.3.3 fails to start

### ISSUE THREAD

**Issue by Kernald** [buildbuddy-io/buildbuddy#872]
Using the Docker image with a quite simple config file:

```
app:
  build_buddy_url: "https://redacted.com"
    #cache_api_url: "grpc://redacted.com:9092"

database:
  data_source: "sqlite3:///buildbuddy.db"

storage:
  ttl_seconds: 604800  # One week
  chunk_file_size_bytes: 3000000  # 3 MB
  disk:
    root_directory: /storage

cache:
  max_size_bytes: 10000000000  # 10 GB
  disk:
    root_directory: /cache

```

It crashes on start-up:
```
2021/08/12 04:50:31.730 INF BuildBuddy v2.3.4 (d85aa1d0d447d980c4c483a7fbe9fe71fc292fd6) compiled with go1.16.2
2021/08/12 04:50:31.730 INF Reading buildbuddy config from '/config.yaml'
2021/08/12 04:50:31.732 INF Auto-migrating DB
2021/08/12 04:50:31.744 INF Cache: BuildBuddy cache API enabled!
2021/08/12 04:50:31.765 INF Finished initializing disk cache partition "default" at "/cache". Current size: 626605297 (max: 10000000000) bytes
2021/08/12 04:50:32.437 FTL Error reading app bundle hash: open sha.sum: file does not exist
```

Every version above 2.3.3 that I tried showed the same issue. I tried clearing all the storage/db/cache data, with the same result.

**Comment by tylerwilliams** [comment]
Hey @Kernald, thanks for filing this. We recently added support for reading static files from a CDN and I suspect it's causing this problem. Will take a look tomorrow and let you know when we've got a fix ready.

**Comment by siggisim** [comment]
Hey @Kernald - we've fixed this in https://github.com/buildbuddy-io/buildbuddy/pull/874

We've cut a [new release](https://github.com/buildbuddy-io/buildbuddy/releases/tag/v2.4.10) `2.4.10` and pushed new docker images that include the fix.

Thanks again for reporting!

**Comment by Kernald** [comment]
Thanks for the quick fix! I just tested 2.4.10, works perfectly for me as well :-)
